lib/components/molecules/StickyHeader/StickyHeader.js

```diff
@@ -0,0 +1,40 @@
+import React, { useState, useRef, useEffect, Fragment } from 'react';
+import type, { Node } from 'react';
+import styled from 'styled-components';
+import classnames from 'classnames';
+import styles from './StickyHeader.style';
+import './StickyHeader.css';
+
+type Props = {
+  children: Node,
+};
+
+const StickyHeader = ({ children, className }: Props): Node => {
+  const [isSticky, setSticky] = useState(false);
+  const ref = useRef(null);
+  const handleScroll = () => {
+    setSticky(ref.current.getBoundingClientRect().top <= 0);
+  };
+
+  useEffect(() => {
+    window.addEventListener('scroll', handleScroll);
+
+    return () => {
+      window.removeEventListener('scroll', () => handleScroll);
+    };
+  }, []);
+
+  return (
+    <Fragment>
```

Comment:
What the use of <Fragment> here you already had a wrapper Div

Reply:
I missed that there. Fixed.

Reply:
Fixed.